### ISSUE TITLE
image element for Q's with responses & delivered Q's

### ISSUE THREAD

**Issue by davidmooreppf** [opengovernment/askthem-legacy#354]
Need prominent images above the fold for Q's with responses, viz. ::  

http://www.askthem.io/ny-brooklyn/questions/52f7eba9ec8d83370b000002

... could be a 'seal' that says "Response!" (Probably better not go with "answered".) 

Or "Official Response" - probably better not go with "verified response", as it's redundant. 

---

... and something for Q's that have been delivered, could be an email icon that says "Delivered!" 

---

In short term let's see about going with some bold copy up top that page, @walter? 

cc @reville for further design help 

![answered_q_askthem_skitch](https://f.cloud.github.com/assets/116819/2151301/1c908018-940a-11e3-8c71-67ec5fcbde44.png)
